js/flightlog_fielddefs.js

```diff
@@ -248,3 +250,16 @@ var
         "LANDING",
         "LANDED"
     ]);
+
+function adjustFieldDefsList(firmwareType, firmwareVersion) {
+    if((firmwareType == FIRMWARE_TYPE_BETAFLIGHT) && semver.gte(firmwareVersion, '3.3.0')) {
+        DEBUG_MODE = DEBUG_MODE_COMPLETE.slice(0);
+        DEBUG_MODE.splice(DEBUG_MODE.indexOf('MIXER'),1);
```

Comment:
Having a space after comma (just like in natural language) normally makes things a bit easier to read.

Reply:
True, I didn't notice it. I will change it tomorrow morning.